pkg/cmd/install.go

```diff
@@ -799,10 +800,22 @@ func installCommand() *cli.Command {
 					return err // we want the user to see the error message without a prefix
 				}
 			}
+
+			if err := maybePromptForAppUpdate(c, prompts.New(), license); err != nil {
+				if errors.Is(err, ErrNothingElseToAdd) {
+					metrics.ReportApplyFinished(c, err)
+					return err
+				}
+				// If we get an error other than ErrNothingElseToAdd, we warn and continue as
+				// this check is not critical.
+				logrus.Warnf("Failed to check for newer app versions: %v", err)
```

Comment:
Does this get printed out? IMO we should just log it internally if anything and move on. There's nothing a user can do about this, so it's more likely to confuse them than anything.

Reply:
updated